devel/kcov/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        SimonKagstrom kcov 37 v
+github.tarball_from archive
+
+categories          devel
+license             GPL-2
+platforms           darwin
+maintainers         nomaintainer
+
+description         coverage report generator
+long_description    \
+    Kcov is a code coverage tester for compiled programs, Python scripts and \
+    shell scripts. It allows collecting code coverage information from \
+    executables without special command-line arguments, and continuously \
+    produces output from long-running applications.
+homepage            https://simonkagstrom.github.com/kcov/index.html
+
+checksums           rmd160  fd849129563ad2ea661b0185988376bf44006eea \
+                    sha256  a136e3dddf850a8b006509f49cc75383cd44662169e9fec996ec8cc616824dcc \
+                    size    305893
+
+depends_build-append port:cmake port:pkgconfig port:python38
```

Comment:
The cmake portgroup should already be adding a dependency on cmake and so it should be removed here.

Reply:
ack